### PR TITLE
fix flaky configapi fuzzing for SA grant method

### PR DESCRIPTION
**Pull request by deads2k**
Fixes https://github.com/openshift/origin/issues/8947

@kargakis ptal

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

File: pkg/cmd/server/api/serialization_test.go [hunk 1]
```diff
@@ -203,6 +203,12 @@ func fuzzInternalObject(t *testing.T, forVersion unversioned.GroupVersion, item
 				obj.NodeSelectorLabelBlacklist = []string{"kubernetes.io/hostname"}
 			}
 		},
+		func(obj *configapi.GrantConfig, c fuzz.Continue) {
+			c.FuzzNoCustom(obj)
+			if len(obj.ServiceAccountMethod) == 0 {
+				obj.ServiceAccountMethod = "prompt"
```
Comment (deads2k) [inline pkg/cmd/server/api/serialization_test.go]: This follows the defaulting rule.  Turns out that the fuzzer set values for each of the fuzzing runs while it was in the merge queue, so it round tripped fine.

**Comment by deads2k** [comment]
@csrwng you're on eastern time, this one is problematic, can you look?

**Comment by pweil-** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

**Comment by kargakis** [comment]
LGTM

**Pull request merged**